services/minespace-web/src/components/pages/ProjectSummaryPage.js

```diff
@@ -1,69 +1,157 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
 import { bindActionCreators } from "redux";
-import { Link } from "react-router-dom";
-import { getFormValues } from "redux-form";
-import { Row, Col, Typography } from "antd";
-import { CaretLeftOutlined } from "@ant-design/icons";
+import { flattenObject, formatUrlToUpperCaseString } from "@common/utils/helpers";
+import { Link, Prompt } from "react-router-dom";
+import { submit, formValueSelector, getFormSyncErrors, reset, touch } from "redux-form";
+import { Row, Col, Typography, Tabs, Divider } from "antd";
+import { ArrowLeftOutlined } from "@ant-design/icons";
 import PropTypes from "prop-types";
-import { getProjectSummary } from "@common/selectors/projectSummarySelectors";
-import { getProjectSummaryDocumentTypesHash } from "@common/selectors/staticContentSelectors";
+import { getMines } from "@common/selectors/mineSelectors";
+import {
+  getProjectSummary,
+  getFormattedProjectSummary,
+} from "@common/selectors/projectSummarySelectors";
+import {
+  getProjectSummaryDocumentTypesHash,
+  getProjectSummaryAuthorizationTypesArray,
+} from "@common/selectors/staticContentSelectors";
 import {
   createProjectSummary,
   fetchProjectSummaryById,
   updateProjectSummary,
 } from "@common/actionCreators/projectSummaryActionCreator";
+import { fetchMineRecordById } from "@common/actionCreators/mineActionCreator";
 import * as FORM from "@/constants/forms";
 import Loading from "@/components/common/Loading";
-import { EDIT_PROJECT_SUMMARY, MINE_DASHBOARD } from "@/constants/routes";
+import { EDIT_PROJECT_SUMMARY, MINE_DASHBOARD, ADD_PROJECT_SUMMARY } from "@/constants/routes";
 import CustomPropTypes from "@/customPropTypes";
-import ProjectSummaryForm from "@/components/Forms/projectSummaries/AddEditProjectSummaryForm";
+import ProjectSummaryForm from "@/components/Forms/projectSummaries/ProjectSummaryForm";
 
 const propTypes = {
-  formValues: PropTypes.shape({
-    project_summary_date: PropTypes.string,
-    project_summary_description: PropTypes.string,
-    documents: PropTypes.arrayOf(PropTypes.object),
-  }),
+  mines: PropTypes.arrayOf(CustomPropTypes.mine).isRequired,
   projectSummary: CustomPropTypes.projectSummary,
   fetchProjectSummaryById: PropTypes.func.isRequired,
   createProjectSummary: PropTypes.func.isRequired,
   updateProjectSummary: PropTypes.func.isRequired,
+  fetchMineRecordById: PropTypes.func.isRequired,
   projectSummaryDocumentTypesHash: PropTypes.objectOf(PropTypes.string).isRequired,
   match: PropTypes.shape({
     params: {
       mineGuid: PropTypes.string,
     },
   }).isRequired,
-  history: PropTypes.shape({ push: PropTypes.func }).isRequired,
+  history: PropTypes.shape({ push: PropTypes.func, replace: PropTypes.func }).isRequired,
+  submit: PropTypes.func.isRequired,
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
just making sure these weren't missed and actually serve a purpose to override the linting.